### PR TITLE
Makes tooltip directive

### PR DESCRIPTION
**Pull request by Kristyn-Mimura**
# Ticket Link

[GROUPINGS-1325](https://uhawaii.atlassian.net/browse/GROUPINGS-1325)

# List of squashed commits

- Makes jasmine test for tooltip directive 
- Replaces all tooltips with the directive 
- Adds jsdoc summary to directive

# Test Checklist

- [X] Unit Tests Passed:
- [X] Jasmine Tests Passed:
- [X] General Visual Inspection:

### CODE REVIEW

**Review by mhodgesatuh — APPROVED** [review]
Glad to see this standardization go into place, and to see that it comes with a unit test.

File: src/main/resources/static/javascript/mainApp/app.directives.js [hunk 1]
```diff
@@ -37,3 +37,30 @@ UHGroupingsApp.directive("tooltipOnTruncate", function () {
...
+                $(element).tooltip("toggle");
+            });
+            element.on("mouseout", function () {
+                $(element).tooltip("dispose");
+            });
+
+        }
+
+        // element.attr("data-placement", attrs["tooltipPosition"]);
+        // this line will make the tooltip nonformatted apparently
```
Comment (JorWo) [inline src/main/resources/static/javascript/mainApp/app.directives.js:63]: Could you add on to this comment "use Bootstrap's data-placement attribute to change placement of tooltip instead"

**Comment by JorWo** [comment]
Sorry, one more thing, I realize the comments would be better as a jsdoc at the top of the directive. Make a summary and list of parameters the directive can take in the jsdoc instead.

**Comment by Kristyn-Mimura** [comment]
> Sorry, one more thing, I realize the comments would be better as a jsdoc at the top of the directive. Make a summary and list of parameters the directive can take in the jsdoc instead.

I'm sorry, what do you mean by parameters?

**Comment by JorWo** [comment]
> > Sorry, one more thing, I realize the comments would be better as a jsdoc at the top of the directive. Make a summary and list of parameters the directive can take in the jsdoc instead.
> 
> I'm sorry, what do you mean by parameters?

Like how tooltip can be on its own or as tooltip="text". Also how you have to use data-placement to set the position.

**Comment by JorWo** [comment]
The jsdoc looks great.

**Review by JorWo — APPROVED** [review]

**Pull request merged**